lib/checks/visibility/hidden-content.js

```diff
@@ -1,13 +1,16 @@
-let styles = window.getComputedStyle(node);
+const whitelist = ['SCRIPT', 'HEAD', 'TITLE', 'NOSCRIPT', 'STYLE', 'TEMPLATE'];
+if (
```

Comment:
This is a weird style? I've been writing multi-line if's like this since forever. The line was getting a bit long so I wrapped it.